subjects/bezero/README.md

```diff
@@ -0,0 +1,40 @@
+## be-zero
```

Comment:
The title should be "bezero"

Reply:
I guess not necessary. 
I just separate two word with '-' like kebab-case.
The title could be Bezero , bezero or be-zero

Reply:
Fixed